lumibot/tools/yahoo_helper.py

```diff
@@ -135,28 +135,11 @@ def process_df(df, asset_info=None):
     # ===================Data download method=============================
 
     @staticmethod
```

Comment:
This function no longer returns any useful info.  Is it intentional?  Should it just be removed?